tensorboard/plugins/graph/tf_graph_app/tf-graph-app.html

```diff
@@ -176,22 +176,36 @@
   _updateHeight: function() {
     this.$$('.container').style.height = this.height + 'px';
   },
-  _updateGraph: function() {
-    if (this.pbtxtFileLocation) {
-      // Fetch a pbtxt file. The fetching will be part of the loading sequence.
-      this.$.loader.datasets = [{
-        // Just name the dataset based on the file location.
-        "name": this.pbtxtFileLocation,
-        "path": this.pbtxtFileLocation,
-      }];
-      this.$.loader.set('selectedDataset', 0);
-    } else if (this.pbtxt) {
+  _fit: function() {
+    this.$$('#graphboard').fit();
```

Comment:
This seems to be new - what does this do?  I also don't see it invoked anywhere?